lib/main.dart

```diff
@@ -116,26 +119,27 @@ class _AppState extends State<App> {
       providers: [
         ChangeNotifierProvider(create: (context) => UserModel()),
         ChangeNotifierProvider(create: (context) {
-          final model = ActivityFeedModel.fromJson(
-              jsonDecode(widget.prefs.getString("activity_feed") ?? "{}"));
+          final model = ActivityFeedModel.fromJson(jsonDecode(widget.prefs
+              .getString("activity_feed", defaultValue: '{}')
+              .getValue()));
           return model
             ..addListener(() {
               widget.prefs
                   .setString("activity_feed", jsonEncode(model.toJson()));
             });
         }),
         ChangeNotifierProvider(create: (context) {
-          final model = LayoutModel.fromJson(
-              jsonDecode(widget.prefs.getString("layout") ?? "{}"));
+          final model = LayoutModel.fromJson(jsonDecode(
+              widget.prefs.getString("layout", defaultValue: '').getValue()));
```

Comment:
`jsonDecode()` will fail here, the `fromJson` call requires an empty object I think?